agent/config.go

```diff
@@ -625,6 +625,10 @@ type Config struct {
 	// true, we ignore the leave, and rejoin the cluster on start.
 	RejoinAfterLeave bool `mapstructure:"rejoin_after_leave"`
 
+	// CheckEnableExec controls whether health checks which execute scripts
+	// are enabled. This includes regular script checks and Docker checks.
+	CheckEnableExec bool `mapstructure:"check_enable_exec"`
```

Comment:
All other toggles are `EnableXXX` or `DisableXXX`. We also have a `DisableRemoteExec` flag which seems to be related to events but I'm not sure whether it relates to this as well. I suggest `EnableScriptChecks` (or `Check` singular) instead unless you have other checks in mind which involve execution of external commands.